test/test_fx.py

```diff
@@ -2812,7 +2812,45 @@ def f(b, a):
         nf = symbolic_trace(nf)
         self.assertEqual(nf(**val), f(**val))
 
+    def test_node_normalize_arguments_use_kwargs(self):
+        class Tracer(torch.fx.Tracer):
+            def is_leaf_module(self, m, module_qualified_name):
+                return False
+
+        foo = torch.nn.Linear(4, 4)
+        trace = Tracer().trace(foo)
+        node = list(trace.nodes)[3]
```

Comment:
Here and below, we shouldn't make an assumption about the absolute position of the node of interest in the node list, as this may change due to unrelated changes in IR.

Instead, let's do something like this:
```
found_node = False
for node in trace.nodes:
  if node.target == torch.nn.functional.linear:
    found_node = True
    sig = inspect.signature(node.target)
    expected_result = {k: v for (k, v) in zip(sig.parameters, node.args)}
    expected_result.update(node.kwargs)
    self.assertEqual(output.kwargs, expected_result)
    self.assertEqual(len(output.args), 0)

self.assertTrue(found_node)
```